ocaml/xapi/helpers.ml

```diff
@@ -440,10 +440,13 @@ type boot_method =
 (* Note: the reason it's OK to trap exceptions and return false is that -- an exn will only happen if the pool record
```

Comment:
You should probably move this comment below, where you define `rolling_upgrade_in_progress`

Reply:
Ah, good catch thanks!